octue/runner.py

```diff
@@ -48,9 +49,16 @@ def __init__(
         show_twined_logs=False,
     ):
         """ Constructor for the Runner class. """
+        # Store the log level (same log level used for all analyses)
+        self._log_level = log_level
+        self.handler = handler
+
+        if show_twined_logs:
+            apply_log_handler(logger=package_logger, handler=self.handler)
 
         # Ensure the twine is present and instantiate it
-        self.twine = Twine(source=twine)
+        self.twine = twine if isinstance(twine, Twine) else Twine(source=twine)
```

Comment:
We need to think about inline if as a company style policy

Reply:
Agreed. I'll avoid one-line if statements apart from for setting default mutable arguments.